autoload/which_key.vim

```diff
@@ -115,31 +115,37 @@ function! s:merge(target, native) " {{{
   let target = a:target
   let native = a:native
 
-  for [k, v] in items(target)
+  for [k, V] in items(target)
 
-    if type(v) == s:TYPE.dict && has_key(native, k)
+    " Support a `Dictionary-function` for on-the-fly mappings
+    if type(V) ==# s:TYPE.funcref
```

Comment:
`type()` returns a Number, so just `==` is fine.

Reply:
Whoops. Fixed.